src/main/kotlin/uk/gov/justice/digital/hmpps/prison/visitbooker/registry/dto/enums/RegisterPrisonerValidationError.kt

```
@@ -0,0 +1,11 @@
+package uk.gov.justice.digital.hmpps.prison.visitbooker.registry.dto.enums
+
+enum class RegisterPrisonerValidationError {
+  BOOKER_ALREADY_HAS_A_PRISONER,
```

BOOKER_ALREADY_HAS_A_PRISONER - is this an issue? Did we want to limit them to 1 prisoner for now? If so, maybe put a comment / TODO about needing to remove this when the 1-prisoner per booker limit is removed

---

will add a TODO to remove it later. The ticket says this - 
We need to do a check initially on this endpoint, if a booker already has an active prisoner, we fail fast with a 422 and a validation message / enum of “active prisoner already found” or something like that.